server/bleep/src/indexes/file.rs

```diff
@@ -117,9 +114,10 @@ impl Indexable for File {
 
                 let workload = Workload {
                     repo_disk_path: &repo.disk_path,
-                    repo_ref: reporef.to_string(),
+                    repo_ref_str: reporef.to_string(),
                     repo_name: &repo_name,
                     file_filter: &file_filter,
+                    repo_ref: reporef,
```

Comment:
do we need these both? `workload.repo_ref_str` is deriavable from `workload.repo_ref.to_string()` no?